### ISSUE TITLE
Is there a python client

### ISSUE THREAD

**Issue by mezongzi** [tus/tus.io#23]
Is there a python client?

**Comment by mezongzi** [comment]
Is there a python client?

**Comment by kvz** [comment]
Hi, yes as can be seen on [the implementations page](http://tus.io/implementations.html) one of our core members @vayam wrote a client library. @codeeply wrote a server implementation.

These are not compatible yet with our [upcoming 1.0 release](https://github.com/tus/tus-resumable-upload-protocol/blob/1.0/protocol.md) though.

If you - or anybody reading this - would like to help with that, we shoot you an invite to our Slack channel and discuss things further (we're considering moving to gitter to have an easier onboarding process, but for now are still hanging out there)

**Comment by JokerQyou** [comment]
I have to say that the server implementation provided by @codeeply is no longer available, the repo is now a 404 page. :(

**Comment by kvz** [comment]
Alright thanks for the headsup, I removed it from our implementation page for now in bc76e3cee84a7280b35ad42adb5ab6173cb76030

**Comment by JokerQyou** [comment]
Umm... By looking through the link you provided, I see the version of us protocol is `1.0`, and it's done on 2014-01-26; But according to the `http://tus.io` website, the version of us protocol is `0.2.2`, and published on the same date. Which one is the right one?

**Comment by kvz** [comment]
Right, 1.0 is our next version. We're in the process of finalizing it. We're still waiting on a final round of review by a few google engineers, but it probably won't change much, which is why implementors already started making their projects compatible with the 1.0 pull request - but it's not the main protocol on tus.io yet.

**Comment by kvz** [comment]
Working on a blog post with Project Status that we'll publish soon.

**Comment by kvz** [comment]
Closing in favor of https://github.com/tus/tus-resumable-upload-protocol/issues/67
